cylc/flow/network/scan.py

```diff
@@ -36,7 +37,6 @@
 MSG_TIMEOUT = "TIMEOUT"
 SLEEP_INTERVAL = 0.01
 
-
```

Comment:
This line removed caused `pycodestyle` to fail again 😟

Could you undo this change, please?

It woukf be great if you could squash your commits too (though I can do it later manually otherwise)